javascript/vue/README.md

```diff
@@ -1,9 +1,64 @@
 # Vue Guide
 
-> TODO: add style guide
-
-Until then please refer to these guides:
+This styleguide is a work-in-progress, documenting how Cloud Four prefers to write Vue. As we continue to expand it, please refer to these guides for additional best practices:
 
 * [Official Vue Style Guide](https://vuejs.org/v2/style-guide/)
 * [Deverus Vue Style Guide](https://gist.github.com/brianboyko/91fdfb492071e743e389d84eee002342)
     * [Erik reviews the Deverus Style Guide](https://www.youtube.com/watch?v=38XnZ3EJqYQ)
+
+## Destructuring in Templates
+
+Try to avoid destructuring in HTML templates, to make it obvious where a particular variable is coming from. Since HTML templates have access to props and local variables in `v-for` loops, clarity is favored over brevity in this case.
+
+**Bad**
```

Comment:
I hesitate to say this is "bad" because I feel it puts down anyone who writes code this way. This decision is more of a personal choice we agreed upon vs a "bad" choice to make. Could it read something else less negative? Perhaps something like "Not preferred" or something like:

### Instead of this:
```js
// code block
```
### We prefer this:
```js
// code block
```

Reply:
That's fair — I hesitated to use bad/good. I stuck with it because that's the standard in all the code style guides I've seen. But that doesn't mean it's right! I'll update my examples.